### PR TITLE
Finetune SAM using SamLoRA

### PR DESCRIPTION
**Pull request by kapil-varshney**
Adds SamLoRA guide.

### CODE REVIEW

**Comment by kapil-varshney** [comment]
@BP-Ent Can you please review this guide?

**Comment by priyankatuteja** [comment]
@BP-Ent  Could you help review this

File: guide/14-deep-learning/finetune-sam-using-samlora.ipynb [hunk 1]
```diff
@@ -0,0 +1,243 @@
+{
```
Comment (BP-Ent) [inline guide/14-deep-learning/finetune-sam-using-samlora.ipynb:1]: <p>Are these strings supposed to exist like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1801/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (kapil-varshney) [reply guide/14-deep-learning/finetune-sam-using-samlora.ipynb:1]: <p>Are you talking about the image encoder string. It renders as an image in the notebook.</p>

**Comment by kapil-varshney** [comment]
Made suggested changes.
@priyankatuteja Can you please approve and merge this guide?

**Comment by priyankatuteja** [comment]
raise pR on next branch

**Pull request closed without merging**